### PR TITLE
Created de_de.json

### PR DESCRIPTION
**Pull request by offron**
Line 11: ` "item.plane.plane_warped": "Verzogenes Flugzeug",` ---is this the correct translation?
Line 14:  `"item.plane.cargo_plane_dark_oak": "Dunkles Eichen Fracht Flugzeug", `  ---not sure about capitalization

### CODE REVIEW

File: src/main/resources/assets/plane/lang/de_de.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  "item.plane.bush_plane_oak": "Eichen Bucsh Flugzeug",
+  "item.plane.bush_plane_dark_oak": "Dunkles Eichen Busch Flugzeug",
+  "item.plane.bush_plane_spruce": "Fichten Busch Flugzeug",
+  "item.plane.bush_plane_acacia": "Akazien Busch Flugzeug",
+  "item.plane.bush_plane_jungle": "Tropen Busch Flugzeug",
+  "item.plane.bush_plane_birch": "Birken Busch Flugzeug",
+  "item.plane.bush_plane_warped": "Verzogenes Bush Flugzeug",
+  "item.plane.bush_plane_crimson": "Purpur Busch Flugzeug",
+  "item.plane.wrench": "Flugzeug Schraubenschlüssel",
+  "item.plane.plane_engine": "Flügzeug Motor",
```
Comment (henkelmax) [inline src/main/resources/assets/plane/lang/de_de.json:30]: `Flügzeug`?

File: src/main/resources/assets/plane/lang/de_de.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{
+  "entity.plane.plane": "Plane",
```
Comment (henkelmax) [inline src/main/resources/assets/plane/lang/de_de.json:2]: Not translated

File: src/main/resources/assets/plane/lang/de_de.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  "entity.plane.cargo_plane": "Fracht Flugzeug",
+  "entity.plane.bush_plane": "Busch Flugzeug",
+  "item.plane.plane_oak": "Eichen Flugzeug",
+  "item.plane.plane_dark_oak": "Dunkles Eichen Flugzeug", 
+  "item.plane.plane_spruce": "Fichten Flugzeug",
+  "item.plane.plane_acacia": "Akazien Flugzeug",
+  "item.plane.plane_jungle": "Tropen Flugzeug",
+  "item.plane.plane_birch": "Birken Flugzeug",
+  "item.plane.plane_warped": "Verzogenes Flugzeug",
```
Comment (henkelmax) [inline src/main/resources/assets/plane/lang/de_de.json:11]: Please look at minecrafts german translation for the nether wood types

File: src/main/resources/assets/plane/lang/de_de.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  "item.plane.plane_crimson": "Purpur Flugzeug",
+  "item.plane.cargo_plane_oak": "Eichen Fracht Flugzeug",
```
Comment (henkelmax) [inline src/main/resources/assets/plane/lang/de_de.json:13]: `Frachtflugzeug`, `Buschflugzeug`

File: src/main/resources/assets/plane/lang/de_de.json [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  "item.plane.bush_plane_spruce": "Fichten Busch Flugzeug",
+  "item.plane.bush_plane_acacia": "Akazien Busch Flugzeug",
+  "item.plane.bush_plane_jungle": "Tropen Busch Flugzeug",
+  "item.plane.bush_plane_birch": "Birken Busch Flugzeug",
+  "item.plane.bush_plane_warped": "Verzogenes Bush Flugzeug",
+  "item.plane.bush_plane_crimson": "Purpur Busch Flugzeug",
+  "item.plane.wrench": "Flugzeug Schraubenschlüssel",
+  "item.plane.plane_engine": "Flügzeug Motor",
+  "item.plane.plane_wheel": "Flugzeug Rad",
+  "item.plane.diamond_reinforced_iron": "Diamanten verstärktes Eisen",
```
Comment (henkelmax) [inline src/main/resources/assets/plane/lang/de_de.json:32]: `Diamantverstärktes ...`

File: src/main/resources/assets/plane/lang/de_de.json [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+  "gui.plane.fuel": "Benzin",
```
Comment (henkelmax) [inline src/main/resources/assets/plane/lang/de_de.json:34]: `Kraftstoff`/`Treibstoff`

File: src/main/resources/assets/plane/lang/de_de.json [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+  "gui.plane.damage": "Schaden",
+  "gui.plane.throttle": "Drossel",
+  "gui.plane.cargo_inventory": "Fracht Inventar",
+
+  "tooltip.plane.fuel": "Benzin: %s mb",
+  "tooltip.plane.damage": "Schaden: %s %%",
+  "tooltip.plane.throttle": "Drossel: %s %%",
+  "tooltip.plane.vertical_speed": "Geschwindigkeit: %s",
```
Comment (henkelmax) [inline src/main/resources/assets/plane/lang/de_de.json:42]: This is the vertical speed

File: src/main/resources/assets/plane/lang/de_de.json [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+  "category.plane": "Plane",
+
+  "key.plane": "Flugzeug GUI",
+  "key.plane_add_thrust": "Mehr Schub",
+  "key.plane_remove_thrust": "Weniger Schub",
+  "key.plane_left": "Links",
+  "key.plane_right": "Rechts",
+  "key.plane_up": "Hoch",
+  "key.plane_down": "Runter",
+  "key.plane_start": "Start",
```
Comment (henkelmax) [inline src/main/resources/assets/plane/lang/de_de.json:59]: `Starten`

**Review by henkelmax — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/plane/lang/de_de.json (hunk 2 above)
Comment (offron) [reply src/main/resources/assets/plane/lang/de_de.json:2]: Wasnt sure if this was the mod name. (misstake i did last time)

**Comment by henkelmax** [comment]
Thank you!

**Pull request merged**